linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/group.yml

```diff
@@ -2,5 +2,11 @@ documentation_complete: true
 
 title: 'Ensure Proper Configuration of Log Files'
 
-description: "The file <tt>/etc/rsyslog.conf</tt> controls where log message are written.\nThese are controlled by lines called <i>rules</i>, which consist of a\n<i>selector</i> and an <i>action</i>.\nThese rules are often customized depending on the role of the system, the\nrequirements of the environment, and whatever may enable\nthe administrator to most effectively make use of log data.\nThe default rules in Red Hat Enterprise Linux 7 are:\n<pre>*.info;mail.none;authpriv.none;cron.none                /var/log/messages\nauthpriv.*                                              /var/log/secure\nmail.*                                                  -/var/log/maillog\ncron.*                                                  /var/log/cron\n*.emerg                                                 *\nuucp,news.crit                                          /var/log/spooler\nlocal7.*                                                /var/log/boot.log</pre>\nSee the man page <tt>rsyslog.conf(5)</tt>\
+description: "The file <tt>/etc/rsyslog.conf</tt> controls where log message are written.\nThese are controlled by lines called <i>rules</i>, which consist of a\n<i>selector</i> and an <i>action</i>.\nThese rules are often customized depending on the role of the system, the\nrequirements of the environment, and whatever may enable\nthe administrator to most effectively make use of log data.
```

Comment:
I'm looking at converting this from a single string (`description: "<some_text>"`) to a block quote-style string:

```
description: |-
  <some longer text with "\n" evaluated>
```

So the edited original would look like:

```
description: |-
    The file <tt>/etc/rsyslog.conf</tt> controls where log message are written.
    These are controlled by lines called <i>rules</i>, which consist of a
    <i>selector</i> and an <i>action</i>.
    These rules are often customized depending on the role of the system, the
    requirements of the environment, and whatever may enable
    the administrator to most effectively make use of log data.
    The default rules in Red Hat Enterprise Linux 7 are:
    <pre>*.info;mail.none;authpriv.none;cron.none                /var/log/messages
    authpriv.*                                              /var/log/secure
    mail.*                                                  -/var/log/maillog
    cron.*                                                  /var/log/cron
    *.emerg                                                 *
    uucp,news.crit                                          /var/log/spooler
    local7.*                                                /var/log/boot.log</pre>
    See the man page <tt>rsyslog.conf(5)</tt>    \ for more information.
    <i>Note that the <tt>rsyslog</tt> daemon can be configured to use a timestamp format that
    some log processing programs may not understand. If this occurs, 
    edit the file <tt>/etc/rsyslog.conf</tt> and add or edit the following line:</i>
    <pre>$ ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat</pre>
```

This would allow us to edit only the center text. Further, I'd propose using our jinja macro `{{{ full_name }}}`  instead of doing the conditional around product. This'll help me in the `debian8` as well; that way, we can condition on the product only for the contents of the defaults.

What do you think?

Reply:
I totally agree with you, multi-line description is much easier to maintain including jinja macro usage. 
The reason I didn't use full_name here is that this is a group file which could be included in generated guide for other products and just replacing product name might confuse readers: assume reader noticing target platform in description but following set of RHEL7 related files is different (it should be updated as well). Thus I left existing RHEL7 as a default example. Let me know if I was wrong and just can use full_name instead :)